### DISCUSSION TITLE
About Display and Share issue

### DISCUSSION THREAD

**Discussion by top58525785** [frangoteam/FUXA#379] [Ideas]
HI FUXA TEAM:
This is a very good SCADA product. 
My question is how to show and share(Password,or Login and so on) it on different  page if multiple different scenarios are created?
Till now ,it is only dispaly one project one time right?
Thanks
Jason

**Comment by unocelli** [comment]
Hi,
You have to enable [authentication](https://github.com/frangoteam/FUXA/wiki/Settings) and create pages for your scenarios and than for example configure the [navigation](https://github.com/frangoteam/FUXA/wiki/HowTo-UI-Layout) with authorization

**Comment by top58525785** [comment]
thanks for your reply。
my question is how to build different projects and show page in different url？



发自我的iPhone


------------------ Original ------------------
From: Umberto Nocelli ***@***.***&gt;
Date: Wed,Jun 1,2022 3:35 AM
To: frangoteam/FUXA ***@***.***&gt;
Cc: top58525785 ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [frangoteam/FUXA] About Display and Share issue (Discussion #379)





 
Hi,
 You have to enable authentication and create pages for your scenarios and than for example configure the navigation with authorization

**Reply by unocelli** [reply]
fuxa works with a project. I think that you can start fuxa from different folder and configure different port.

**Reply by top58525785** [reply]
Thanks Unocelli.  Is there any instruction to tell how to start fuxa using different folder? to deploy again?

**Reply by unocelli** [reply]
I think should work if you [download or clone and install ](https://github.com/frangoteam/FUXA/wiki/Installing-and-Running)the project in every folder and then you change the port in server/_appdata/settings.js (uiPort)
